Fuzzing/LFI/LFI-coyote0x90-linux-etc-files.txt

```diff
@@ -0,0 +1,8314 @@
+/etc/3ddesktop/3ddesktop.conf
```

Comment:
It's better if wordlists have descriptive names, so that people who want to use them don't have to look them up constantly. I think this should be named `LFI-etc-files-of-all-linux-packages.txt`